fe-games/src/components/Review.jsx

```diff
@@ -21,6 +29,42 @@ export function Review() {
     });
   }, [review_id]);
 
+  const upVote = (review_id) => {
+    upvoted ? (vote = -1) : (vote = 1);
+    setCurrentReview((currentReview) => {
+      if (!upvoted) {
+        setHasUpVoted(true);
+        return { ...currentReview, votes: currentReview.votes + vote };
+      } else {
+        setHasUpVoted(false);
+        return { ...currentReview, votes: currentReview.votes + vote };
+      }
+    });
+    setErr(null);
+    changeReviewVotes(review_id, vote).catch((err) => {
+      setErr("Something went wrong, please try again");
+      return { ...currentReview, votes: currentReview.votes + vote };
+    });
+  };
+
+  const downVote = (review_id) => {
```

Comment:
both voting buttons increment the votes, so have a look at your logic here, something needs flipped around so that the down vote decrements the votes